ekip/everykid/templates/base.html

```diff
@@ -1,18 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
 
-{% block head%}
```

Comment:
We might want to put blocks around the <title> tag, so that we can override it later.